frontend/src/components/Landlord/Header.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { ReactElement } from 'react';
+import HeartRating from '../utils/HeartRating';
+import ApartmentPhoto1 from '../../assets/apartment_exterior.png';
```

Comment:
Since we're going to store links to the landlord photos in our database, let's replicate that by adding/using dummy image links from landlordData instead of adding the dummy images to the project